guides/common/modules/proc_changing-the-mirroring-policy-for-a-repository.adoc

```diff
@@ -0,0 +1,13 @@
+[[changing_the_mirroring_policy_for_a_repository]]
+= Changing the Mirroring Policy for a Repository
+
+
```

Comment:
```suggestion
```

Reply:
@maximiliankolb this new ID should have ` _{context}` appended to it also, IIRC?

Reply:
Ah I see your other comment!